src/containers/beerList.js

```diff
@@ -41,11 +41,11 @@ class BeerList extends Component {
         return _.map(this.props.beers, beer => {
             
             return (
-                <tr  key={beer.id}  onClick = {(data) => {this.showDesc(beer.id)}}>
-                <td>{beer.beerName}</td>
-                <td className='theRating'>< RatingsComponent beerRating = {beer.rating}  beerId={beer.id}/></td>
-                <td >{beer.desc}</td>
-                </tr>
+                <div  key={beer.id} className='row'>
+                <div className='col-sm-2 '>{beer.data.beerName}</div>
+                <div className='theRating col-sm-3'>< RatingsComponent beerRating = {beer.data.rating}  beerId={beer.id}/></div>
+                <div className='col-sm-3 '>{beer.data.desc}</div>
```

Comment:
Dont use bootstrap, this project is an exercise, so you shouldn't do shortcuts.

Reply:
alright, I'll remove it